packages/integration/lib/src/widgets/lists/feed/feed_list.dart

```diff
@@ -657,6 +660,7 @@ class _LMFeedListState extends State<LMFeedList> {
           eventName: postViewData.isPinned
               ? LMFeedAnalyticsKeys.postPinned
               : LMFeedAnalyticsKeys.postUnpinned,
+          widgetSource: LMFeedWidgetSource.universalFeed,
```

Comment:
check this also, is it right to assign widgetSurce: LMFeedWidgetSource.universalFeed ?